src/CSF.SRDashboard.Client/Components/SearchErrorComponent.razor.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CSF.SRDashboard.Client.PageValidators;
+using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Components
+{
+    public partial class SearchErrorComponent
+    {
+        [Parameter]
+       public SearchErrorObj searchError { get; set; }
```

Comment:
Capitalize the first word of a public parameter.
```suggestion
       public SearchErrorObj SearchError { get; set; }
```